signac/core/synced_collections/synced_collection.py

```diff
@@ -128,10 +128,19 @@ class SyncedCollection(Collection):
     _validators: List[Callable] = []
     # Backends that support threading should modify this flag.
     _supports_threading: bool = False
+    _LoadSaveType = _LoadAndSave
 
     def __init__(self, parent=None, *args, **kwargs):
-        self._parent = parent
-        self._suspend_sync_ = 0
+        if parent is not None:
+            self._parent = parent._parent if parent._parent is not None else parent
+        else:
+            self._parent = parent
```

Comment:
In addition to the suggestion above, I believe this logic could be shorter (but possibly not clearer -- you can make the call of whether to change this).
```suggestion
        self._parent = getattr(parent, "_parent", None) or parent
```

Reply:
While shorter, I believe this is harder to follow and am skipping this change.